### ISSUE TITLE
WCAG: Miles image block: Text overflowing, alt text, and focus / hover styling

### ISSUE THREAD

**Issue by skaihigh** [miles-no/miles_web_components#61]
There are three issues with this comonent.
- Text is overflowing. See image for example. This issue happens on certain screen widths. It also happens when zooming 200% on the webpage. This is a violation of WCAG requirements, as all content should be available at 400% zoom.
- The image needs a good alt text. "Miles image card image" is not sufficient.
- Another thing that should be considered is to fix the hover/ focus styling on the link in the component.

![Image](https://user-images.githubusercontent.com/1680898/258332664-bb7245fd-8edb-450d-9c77-6d61406a8a36.png)

**Comment by skaihigh** [comment]
* added hover and selected state on button. Button should probably be a component and be injected into the image block via a slot.
